DYNAMIC.md

```diff
@@ -0,0 +1,326 @@
+    void createOrUpdateAllFromJson(String clazz, String json);
+    void createAllFromJson(String clazz, InputStream inputStream) throws IOException;
+    void createOrUpdateAllFromJson(String clazz, InputStream in) throws IOException;
+    void createObjectFromJson(String clazz, JSONObject json);
+    DynamicRealmObject createOrUpdateObjectFromJson(String clazz, JSONObject json);
+    DynamicRealmObject createObjectFromJson(String clazz, String json);
+    DynamicRealmObject createOrUpdateObjectFromJson(String clazz, String json);
+    DynamicRealmObject createObjectFromJson(String clazz, InputStream inputStream) throws IOException;
+    DynamicRealmObject createOrUpdateObjectFromJson(String clazz, InputStream in) throws IOException;
+    void writeCopyTo(File destination) throws IOException;
+    void writeEncryptedCopyTo(File destination, byte[] key) throws IOException;
+    DynamicRealmObject createObject(String clazz);
+    DynamicRealmObject copyToRealm(DynamicRealmObject object);
+    DynamicRealmObject copyToRealmOrUpdate(DynamicRealmObject object);
+    List<DynamicRealmObject> copyToRealm(Iterable<DynamicRealmObject> objects);
+    List<DynamicRealmObject> copyToRealmOrUpdate(Iterable<DynamicRealmObject> objects);
+    RealmQuery<DynamicRealmObject> where(String clazz);
+    RealmResults<DynamicRealmObject> allObjects(String clazz);
+    RealmResults<DynamicRealmObject> allObjectsSorted(String clazz, String fieldName,
+                                                             boolean sortAscending);
+    RealmResults<DynamicRealmObject> allObjectsSorted(String clazz, String fieldName1,
+                                                             boolean sortAscending1, String fieldName2,
+                                                             boolean sortAscending2);
+    RealmResults<DynamicRealmObject> allObjectsSorted(String clazz, String fieldName1,
+                                                             boolean sortAscending1,
+                                                             String fieldName2, boolean sortAscending2,
+                                                             String fieldName3, boolean sortAscending3);
+    RealmResults<DynamicRealmObject> allObjectsSorted(String clazz, String fieldNames[],
+                                                             boolean sortAscending[]);
+    void addChangeListener(RealmChangeListener listener);
+    void removeChangeListener(RealmChangeListener listener);
+    void removeAllChangeListeners();
+    void refresh();
+    void beginTransaction();
+    void commitTransaction();
+    void cancelTransaction();
+    void executeTransaction(DynamicRealm.Transaction transaction);
+    void clear(String clazz);
+    String getPath();
+    RealmConfiguration getConfiguration();
+}
+
+
+*DynamicRealmObject*
+
+public class DynamicRealmObject {
+    boolean getBoolean(String fieldName);
+    int getInt(String fieldName);
```

Comment:
Since we are going to have boxed type, the interface numbers will grow... IMO, make it `Integer getInt(String fieldName)` won't be a good choice, since it will slow down the primitive calls. `getIntBoxed`, this name doesn't look good either.